qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -621,10 +632,8 @@ def _calc_decoupling(pauli_x, pauli_z, qubit_list, min_qubit, num_qubits, cliff)
     circ = QuantumCircuit(num_qubits)
 
     # decouple_cliff is initialized to an identity clifford
-    decouple_cliff = cliff.copy()
-    num_qubits = decouple_cliff.num_qubits
-    decouple_cliff.table.phase = np.zeros(2 * num_qubits)
-    decouple_cliff.table.array = np.eye(2 * num_qubits)
+    num_qubits = cliff.num_qubits
+    decouple_cliff = Clifford(np.eye(2 * num_qubits))
```

Comment:
I think we can pass `verify=False` to `Clifford` for a small speed-up here?

Reply:
In addition, doesn't this a cyclic import?

Reply:
There's lots of cyclic imports in this file from before the deprecation - it's not ideal perhaps, but the imports of the name `Clifford` are protected within the functions, so it should work fine.

Reply:
I agree with your idea, but new feature should be in separate PR.